catt/subs_info.py

```diff
@@ -0,0 +1,51 @@
+import re
+
+import requests
+
+from .error import SubtitlesError
+from .util import create_temp_file
+
+
+class SubsInfo:
+    def __init__(self, subs_url: str, local_ip: str, port: int) -> None:
+        self._subs_url = subs_url
+        self._local_ip = local_ip
+        self.port = port
```

Comment:
Why is `port` not `_port`? (Add a comment explaining, if it's intentional).

Reply:
I was just thinking that making `self._local_ip` public would be superflous, as the local ip is already exposed as StreamInfo.local_ip. But yeah, you're right, it looks strange, so maybe it should just be:
```
self.local_ip = local_ip
self.port = port
```

I agree with all your other comments, and will apply fixes.

Reply:
Ah, I see. Either way is good with me